.drone.yml

```diff
@@ -61,6 +61,14 @@ pipeline:
       branch: master
       event: push
 
+    trigger_deploy_to_dev:
+      image: quay.io/ukhomeofficedigital/drone-trigger:v0.3.0
+      drone_server: https://drone-gitlab.acp.homeoffice.gov.uk/modern-slavery/kube-modern-slavery
```

Comment:
drone server should be https://drone-gitlab.acp.homeoffice.gov.uk - not the full repo path

Reply:
you also need to reference the relevant secret for the gitlab drone token

Reply:
I'd also add when this should triggered, e.g.
```
when:
      branch: master
      event: push
```